daemon/qrexec-daemon.c

```diff
@@ -838,6 +863,13 @@ static int parse_policy_response(
             *target = strdup(current_response + (sizeof("target=") - 1));
             if (*target == NULL)
                 abort();
+        } else if (!strncmp(current_response, "target_uuid=", sizeof("target_uuid=") - 1)) {
+            if (*target_uuid != NULL)
+                goto bad_response;
+            *target_uuid = strdup(current_response + 7);
+            if (*target_uuid == NULL)
+                abort();
+            (*target_uuid)[4] = ':';
```

Comment:
This requires a comment what is happening here.

Reply:
What is happening is that the response is `target_uuid=SOMEUUID`, but we need `uuid:SOMEUUID`.  That is done by adding 7 (to skip past the `target_`) and then overwriting the `=` with `:`.

Reply:
Yes, I know. Put that into a comment.